db/load.ts

```diff
@@ -1,25 +1,49 @@
+import { Client } from 'pg';
 import { Guid } from 'guid-typescript';
-import { Database, constants } from 'bun:sqlite';
+
+// Database connection setup
+const client = new Client({
+  host: 'localhost',
+  port: 5432,
+  database: 'crawler',
+  user: 'admin',
+  password: 'password',
+});
```

Comment:
is this one for testing?

Reply:
Yes, I tried to support test scripts in /db directory. If we dont need them for now, I can remove them from the repo